create_jenkins_job.py

```diff
@@ -152,7 +152,8 @@ def main(argv=None):
             'label_expression': 'linux',
             'shell_type': 'Shell',
             'build_args_default': '--packages-skip-by-dep ros1_bridge --packages-skip ros1_bridge ' + data['build_args_default'],
-            'test_args_default': '--packages-skip-by-dep ros1_bridge --packages-skip ros1_bridge ' + data['test_args_default'],
+            'test_args_default': '--packages-skip-by-dep ros1_bridge --packages-skip ros1_bridge ' + re.sub(
+                r'(--ctest-args -LE )"?([^ "]+)"?', r'\1"(cppcheck|\2)"', data['test_args_default']),
```

Comment:
A typo that led to two spaces after `-LE` wouldn't be caught by this expression, but we control the string here so this seems fine.

Reply:
I made an easy update to the regex to handle this: 9c0e1ab